### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by ShaneKilkelly**
### Description

Update critical dependencies. Including an upgrade to express v4.

#### Related Issues / PRs

- Part of https://github.com/overleaf/issues/issues/2782

### Review

- PR is split up into logically separate commits, could be easier to review on a commit-by-commit basis

#### Potential Impact

- Farily high, this is a very important component of our stack.

#### Manual Testing Performed

- [ ] Check that editing documents works in the browser

#### Who Needs to Know?

@henryoswald

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,11 +26,11 @@
     "coffee-script": "~1.7.0",
     "express": "4.17.1",
     "lodash": "^4.17.13",
-    "logger-sharelatex": "^1.7.0",
-    "metrics-sharelatex": "^2.5.1",
-    "mongojs": "^2.6.0",
+    "logger-sharelatex": "^1.9.0",
+    "metrics-sharelatex": "^2.6.0",
```
Comment (emcsween) [inline package.json]: There are some recent important bugfixes in the metrics module:

```suggestion
    "metrics-sharelatex": "^2.6.2",
```

File: package.json [hunk 2]
```diff
@@ -26,11 +26,11 @@
     "coffee-script": "~1.7.0",
     "express": "4.17.1",
     "lodash": "^4.17.13",
-    "logger-sharelatex": "^1.7.0",
-    "metrics-sharelatex": "^2.5.1",
-    "mongojs": "^2.6.0",
+    "logger-sharelatex": "^1.9.0",
```
Comment (emcsween) [inline package.json]: In the logger module as well:
```suggestion
    "logger-sharelatex": "^1.9.1",
```

**Comment by ShaneKilkelly** [comment]
Good catch! Thanks, I've made the changes.

**Comment by briangough** [comment]
This is unblocked now, the redis migration is complete.

**Pull request merged**